terra-jupyter-aou/tests/aou_smoke_test.ipynb

```diff
@@ -134,80 +226,6 @@
     "## Test that our source control protections are working."
```

Comment:
Everything below here should be deleted.

This file should look like [this](https://github.com/DataBiosphere/terra-docker/blob/a86cb30223f6fc25dc34a6cd8608026eb2d5713e/terra-jupyter-aou/tests/aou_smoke_test.ipynb)

Reply:
oops...didn't push the local change